backend/Persistence/Repositories/UserRepository.cs

```diff
@@ -52,7 +52,8 @@ public async Task<User> Authenticate(string mail, string password)
             var tokenDescriptor = new SecurityTokenDescriptor
             {
                 Subject = new ClaimsIdentity(new Claim[] {
-                    new Claim(ClaimTypes.Email, user.Mail)
+                    new Claim(ClaimTypes.Email, user.Mail),
+                    new Claim(ClaimTypes.Name, user.Mail)
```

Comment:
I am not too familiar (not at all really) with .net core, but is the new claim supposed to be of type Name and still use 'user.Mail'?

Reply:
Since the username is mail I found it logical to to be of both. This makes it easier to retrieve the username like this: var targetUsername = httpContextAccessor.HttpContext.User.Identity.Name, while also letting it be what it is an email address. I did however reflect a bit on this before deciding to do it this way, so if you have any input or thought we could maybe reflect some more.